test/server/content-utils-test.js

```diff
@@ -38,6 +44,37 @@ describe('Content utils', function () {
                     done();
                 });
         });
+
+        it('Should correctly determine the charset from BOM', function (done) {
```

Comment:
It's should be moved to the `charset` tests. And shouldn't we test other parsing scenarios, e.g. meta's?